styles/style.css

```diff
@@ -0,0 +1,109 @@
+
+div {
+    margin: 0px;
+    width: 100vw;
+    height: 100vh;
+    display: grid;
+    grid-template-rows: repeat(10,10%);
+    grid-template-columns: 20%, 60%, 20%;
+}
+
+.portfolio {
```

Comment:
This class is only used inside one `div` element and copies a lot of the same properties that the `div` is given above. The code can be DRY-ed up by removing duplicates.